n3fit/src/n3fit/model_trainer.py

```diff
@@ -922,7 +926,11 @@ def hyperparametrizable(self, params):
             for model in models.values():
                 model.compile(**params["optimizer"])
 
-            passed = self._train_and_fit(models["training"], stopping_object, epochs=epochs,)
+            passed = self._train_and_fit(
+                models["training"],
+                stopping_object,
+                epochs=epochs,
+            )
```

Comment:
I don't think this change is necessary, with the configuration as defined in `pyproject.toml` since it was already less than 100 columns. 

I did notice `black` automatically 'unfolds' inputs like this if there is a trailing comma, so please just remove the last comma.

Reply:
See https://github.com/NNPDF/nnpdf/pull/1828

Reply:
Done.

Reply:
Thanks! Instead of saying done, in principle whenever you make an edit to a line that has been commented on the conversation can simply be marked as resolved - in many cases this will even be done automatically. That way the conversation is folded and the thread is cleaner.

Reply:
Ah you just did ;)